src/js/profile/chart.js

```diff
@@ -196,6 +196,10 @@ export class Chart extends Observable {
               height: { scale: "yscale", band: 1 },
               x: { scale: "xscale", field: { signal: "datatype[Units]" } },
               x2: { scale: "xscale", value: 0 },
+              tooltip: [{
```

Comment:
we need to format the numbers, _Vega-tooltip_ uses it like this and I think we should do the same:
```json
{ "tooltip": {"signal": "{'Unemployment Rate': format(datum.unemp.rate, '0.1%')}"}},
```
so for us this would be:
```js
tooltip: {
  signal: "{'percentage': format(datum.percentage, numberFormat.percentage), 'group': mainGroup, 'count': format(datum.count, numberFormat.value)}"
}
```
the signal will be evaluated by Vega and you can use the signals and datum from Vega.

Reply:
Implemented. I am still getting familiar with vega signals